reference/api/indexes.md

```diff
@@ -64,15 +64,19 @@ Get information about an [index](/learn/core_concepts/indexes.md).
 
 <RouteHighlighter method="POST" route="/indexes"/>
 
-Create an [index](/learn/core_concepts/indexes.md).
+Create an [index](/learn/core_concepts/indexes.md). This route accepts two arguments: `index_uid` and `primaryKey`.
 
-This route takes as parameter an unique `uid` and **optionally** the [primary key](/learn/core_concepts/indexes.md#primary-key).
+`index_uid` is mandatory. It should be a unique string identifying your new index.
```

Comment:
```suggestion
`uid` is mandatory. It should be a unique string identifying your new index.
```

When creating an index resource, the field name for naming the index (his unique identifier) is still named `uid`.

The `202` response does not contain an `index` resource but a summarized `task` resource. 

That's why the `task` resource contains an `indexUid` to give information that a `task` has been enqueued to create that index and will belong to this index.

_**Generally speaking**_, when we design an API resource, we avoid putting its name in the fields that concern it.

On the other hand, it is advisable to contextualize fields linked to another resource by a prefix of the resource name or an object bearing the name of the linked resource. As in the case of a `task` that is linked to an `index`, it could have been: 

```json
{
    "uid": 0, //task uid
    "index": {
        "uid": "movies"
    },
    ...
}
```

But we chose this format since we don't expect to give more information from the `index` related to a `task`.

```json
{
    "uid": 0,
    "indexUid": "movies",
    ...
}
```